types/types.go

```diff
@@ -546,3 +546,44 @@ func (clientTLS *ClientTLS) CreateTLSConfig() (*tls.Config, error) {
 	}
 	return TLSConfig, nil
 }
+
+// Domains parse []Domain
+type Domains []Domain
+
+// Set []Domain
+func (ds *Domains) Set(str string) error {
+	fargs := func(c rune) bool {
+		return c == ',' || c == ';'
+	}
+	// get function
+	slice := strings.FieldsFunc(str, fargs)
+	if len(slice) < 1 {
+		return fmt.Errorf("Parse error ACME.Domain. Imposible to parse %s", str)
```

Comment:
Could you please change the error to `parse error ACME.Domain. Unable to parse %s`